resources/tests/setup_rootfs.sh

```diff
@@ -11,7 +11,14 @@ prepare_fc_rootfs() {
     SSH_DIR="$BUILD_DIR/ssh"
     RESOURCE_DIR="$2"
 
-    packages="udev systemd-sysv openssh-server iproute2 msr-tools"
+    packages="udev systemd-sysv openssh-server iproute2"
+
+    # msr-tools is only supported on x86-64.
+    arch=`uname -m`
```

Comment:
nit: I think `shellcheck` prefers `$()` (it's a bit more readable to me).

```sh
arch=$(uname -m) 
```

Reply:
Yes, you are correct. `arch=$(uname -m)`  will be preferable.